pyfar/dsp/dsp.py

```diff
@@ -1973,10 +1973,9 @@ def average(signal, mode='linear', caxis=None, weights=None, keepdims=False,
                                pyfar.TimeData)):
         raise TypeError(("Input data has to be of type 'Signal', 'TimeData' "
                          "or 'FrequencyData'."))
-    if type(signal) == pyfar.TimeData and mode in ('log_magnitude_zerophase',
-                                                   'magnitude_zerophase',
-                                                   'magnitude_phase',
-                                                   'power',):
+    if type(signal) is pyfar.TimeData and mode in (
+            'log_magnitude_zerophase', 'magnitude_zerophase',
+            'magnitude_phase', 'power',):
```

Comment:
Is the trailing comma necessary?

Reply:
no, but it was already there. I removed it..